x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -37,4 +41,15 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number>> = {
     }),
     schema: schema.number(),
   },
+  [enableComparisonByDefault]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.enableComparisonByDefault', {
+      defaultMessage: 'Enable comparison by Default',
```

Comment:
```suggestion
      defaultMessage: 'Comparison feature',
```

Reply:
The description below works but I felt the title could be better. Not been able to test it locally, but will it say it's enabled by default in the default value section?

Reply:
yes: 

<img width="343" alt="Screen Shot 2021-11-03 at 4 35 11 PM" src="https://user-images.githubusercontent.com/55978943/140188667-c4746efd-c540-497d-8728-6c62a0aba067.png">